src/main/java/software/amazon/cloudformation/proxy/AmazonWebServicesClientProxy.java

```diff
@@ -146,11 +222,11 @@ public ClientT client() {
         }
 
         @Override
-        public <RequestT> Caller<RequestT, ClientT, ModelT, CallbackT> request(Function<ModelT, RequestT> maker) {
+        public <RequestT> Caller<RequestT, ClientT, ModelT, CallbackT> translate(Function<ModelT, RequestT> maker) {
             return new Caller<RequestT, ClientT, ModelT, CallbackT>() {
 
                 @Override
-                public Caller<RequestT, ClientT, ModelT, CallbackT> retry(Delay delay) {
```

Comment:
tbh `retry` is more intuitive for me but that might be just me

Reply:
backoff strategy for delays in a norm. Open to suggestions.